addons/website_event_meet/views/event_event_views.xml

```diff
@@ -25,4 +25,3 @@
     </record>
 
 </data></odoo>
-
```

Comment:
maybe we can remove this to avoid modifying the module in this task? unless runbot is not happy about it?

Reply:
@Odoonan Obviously, thanks.